src/inference/src/ie_core.cpp

```diff
@@ -372,7 +376,7 @@ class CoreImpl : public ie::ICore, public std::enable_shared_from_this<ie::ICore
     }
 
 public:
-    CoreImpl(bool _newAPI) : newAPI(_newAPI) {
+    CoreImpl(bool _newAPI) : executorManagerPtr(executorManager()), newAPI(_newAPI) {
```

Comment:
looks like it's not needed here?

Reply:
I think in case if all plugins are unloaded we should keep the execution manager somewhere.